packages/aws-cdk-lib/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -87,6 +87,16 @@ export interface CommonAutoScalingGroupProps {
    */
   readonly keyName?: string;
 
+  /**
+   * The SSH keypair to grant access to the instance.
+   *
+   * `keyName`, `launchTemplate` and `mixedInstancesPolicy` must not be specified
+   *  when this property is specified
+   *
```

Comment:
```suggestion
   * `launchTemplate` and `mixedInstancesPolicy` must not be specified
   *  when this property is specified.
   *
   * You can either specify `keyPair` or `keyName`, not both.
```
Suggestion for different wording. Can you please update `keyName` documentation as well?

Reply:
Fixed in https://github.com/aws/aws-cdk/pull/29679/commits/4940cd1a993fa6d41a7a0dde06e3feb95735a2c0